`Text` component - Code implementation (#1490)
hashicorp/design-system: website/docs/components/text/partials/accessibility/accessibility.md

## Conformance rating

<!-- Update conformance rating badge with correct status and remove the others -->

> <Doc::Badge @type="success">Conformant</Doc::Badge>

Reviewer: ```suggestion
<Doc::Badge @type="warning">Conditionally Conformant</Doc::Badge>
```

> When used as recommended, there should not be any WCAG conformance issues with this component.
> 
> !!! Info
> 
> As this component is nothing more than a wrapper around generic HTML tags with custom typographic styles applied, any conformance issues that may arise will depend on how this component is used in the consumer’s code, in the same way as normal HTML tags and text would.
> 
> !!!

Reviewer: ```suggestion
Since this component is nothing more than a wrapper around generic HTML tags with custom typographic styles applied, any conformance issues that may arise will depend on how this component is used in the consumer’s code, in the same way as normal HTML tags and text would.
```
